Added close and reopen commands. Fixes #9

rgee0 opened the pull request:
Signed-off-by: rgee0 <richard@technologee.co.uk>

Added the ability for maintainers to close and reopen issues & PRs to address #9 
Also performed minimal update the README.md to include the new commands and add an updated `faas-cli deploy` command.

Tested on one of my own [repos] (https://github.com/rgee0/Test/pull/3) used for testing.

alexellis commented:
Thank you Richard 🤖

alexellis commented:
Can we do the close/open command without a comment in-line?

alexellis commented on README.md:
```diff
@@ -87,11 +89,11 @@ Now, build and deploy Derek:
 **Testing**
 
-Create a label of "no-dco" within every project you want Derek to help you with.
+Try creating a label of "no-dco" within a project you want Derek to help you with.
```
Not sure this line needed to change

rgee0 replied:
I changed that because reading it stepwise it appears as an instruction; that is, in order to enable Derek to manage/monitor a repo the user should manually create a `no-dco` label within that repo.  I think its actually a lead-in to the following statement - the change was meant to move more towards a description of the following action.  

No big deal to me, it can come out.

alexellis commented on commentHandler.go:
```diff
@@ -127,6 +127,41 @@ func handleComment(req types.IssueCommentOuter) {
+
+		if allowed {
+			client, ctx := makeClient()
+			state := "closed"
```
Can we extract closed/open to consts please?

rgee0 commented:
Removed the comment (and colon) from reopen and close commands.

Reverted the testing part of README - please consider earlier comments on reasoning.

Extracted the constants.

Merged the switch cases for reopen & close

alexellis commented:
> please consider earlier comments on reasoning.

I think the comment would just be anything after a new line and would have to work for all commands.

alexellis commented on commentHandler.go:
```diff
@@ -157,6 +186,12 @@ func parse(body string) *types.CommentAction {
 	return &commentAction
 }
 
+func isValidCommand(body string, trigger string) bool {
```
This should be unit-testable. Would you be able to follow up in a new PR?

alexellis commented:
Merging, thank you @rgee0
